### ISSUE TITLE
Position marker on track

### ISSUE THREAD

**Issue by luhrern** [pR0Ps/grafana-trackmap-panel#57]
I have used the TrackMap panel in Grafana eariler with Postgres a source. Used LAT / LON and the representation in teh panel was great. When using this toteher with a timesseries panel with another meaturement and using the "Shared Tooltip" in Grafana a Position Marker on the track appeared.

I have now made a new Grafana (Grana Cloud - same subscription as above) dashboard where I use data from an InfluxDB 1.7 database. I use teh TrackMap panel, and have enabled teh "Shared Tooltip". I have several timeseries panled in the dashboard, and the "red vertical" line follow in all the, but there is no Position Marker in the TrackMap panel.

The TrackMap plugin now is verion 2.1.0.

**Comment by pR0Ps** [comment]
Duplicate of #50

A fix has already been released, just waiting on Grafana to publish it (see #52)
